shared/panels/currency-network-sidepanel/currency-network-sidepanel.tsx

```diff
@@ -1,47 +1,46 @@
 import { CurrencyReactQueryAdapter } from "@/core/application/react-query-adapter/currency";
 
 import { Accordion, AccordionLoading } from "@/design-system/molecules/accordion";
-import { CardTemplate } from "@/design-system/molecules/cards/card-template";
+import { CardTemplate, CardTemplateLoading } from "@/design-system/molecules/cards/card-template";
 
 import { ErrorState } from "@/shared/components/error-state/error-state";
 import { SidePanelBody } from "@/shared/features/side-panels/side-panel-body/side-panel-body";
 import { SidePanelHeader } from "@/shared/features/side-panels/side-panel-header/side-panel-header";
 import { useSidePanel, useSinglePanelData } from "@/shared/features/side-panels/side-panel/side-panel";
 import { CardNetwork } from "@/shared/panels/currency-network-sidepanel/_components/card-network/card-network";
 import { useCurrencyNetworkSidepanel } from "@/shared/panels/currency-network-sidepanel/currency-network-sidepanel.hooks";
+import { useDepositTransactionSidepanel } from "@/shared/panels/deposit-transaction-sidepanel/deposit-transaction-sidepanel.hooks";
 
 export function CurrencyNetworkSidepanel() {
   const { name } = useCurrencyNetworkSidepanel();
   const { Panel } = useSidePanel({ name });
   const { currencyId } = useSinglePanelData<{ currencyId: string }>(name) ?? {
     currencyId: "",
   };
+  const { open: openDepositTransactionSidepanel } = useDepositTransactionSidepanel();
```

Comment:
don't want to pass this handler as an event of currencyNetworkSidePanel ? 

As this we can imagine to have another action on another page ?

You can take exemple of the projectSidePanel and the onGrantClick event